components/input/ColorPicker.js

```diff
@@ -19,21 +19,22 @@ const ColorPicker = ({ text, initialRgbaColor, onColorChange, ...rest }) => {
     };
     const handleChange = (color) => {
         setRgbaColor(color.rgb);
+        if (onChange) {
+            onChange(color);
+        }
     };
 
+    useEffect(() => {
```

Comment:
if it's the "initial" value, you don't need to use `useEffect` here. I don't expect to receive a change from `initialRgbaColor` right?

Reply:
Yeah, for the component as it is it's enough. But when I was building the `ColorPickerWithInput`, if I wanted to use the `ColorPicker` component there, I needed a way for the `Input` component to tell the color chosen in the input field to `ColorPicker`, and the natural way to do it seemed to me through `initialRgbaColor`. That's why I wrote the `useEffect` hook like that. For the current functoinality is not needed, but if we later need a way to interact externally with the color displayed in the color picker, I think doing it through `initialRgbaColor` would make sense.

So it depends on how we want to use the `ColorPicker`, I guess. Let me know what's your preference.

Reply:
I see your point. For now, since we don't plan to use an input and the color picker, and because the color picker includes also a field to change the value manually, remove `useEffect` please.

Reply:
ok, done!